src/docker-client.ts

```diff
@@ -22,6 +23,21 @@ export type Dir = string;
 
 export type TmpFs = { [dir in Dir]: Dir };
 
+export type HealthCheck = {
+  test: string;
```

Comment:
> the option that's slightly different than the original one from Docker.

I took "test" from https://docs.docker.com/engine/api/v1.40/#operation/ContainerCreate.

> Also I guess it might throw some users off that you can't both listen to ports and a health check only a single wait strategy will be used. But I think it should be fine for most cases.

We can add this in the future. Should be easy enough to chain them together.